index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
```

Comment:
#### Nitpick :warning: 
Not is necessary to use "Type" in your links, you only can use if you want work with older browsers. In the new browsers the type of languages come for default. If you want read more information you can see Udacity style guide: http://udacity.github.io/frontend-nanodegree-styleguide/index.html